api/src/main/java/org/pmiops/workbench/exceptions/ExceptionAdvice.java

```diff
@@ -25,19 +25,21 @@
 
   @ExceptionHandler({Exception.class})
   public ResponseEntity<?> serverError(Exception e) {
-    ErrorResponse errorResponse = new ErrorResponse();
-    Integer statusCode = HttpStatus.INTERNAL_SERVER_ERROR.value();
-
+    ErrorResponse errorResponse = WorkbenchException.errorResponse();
```

Comment:
Why a static method in WorkbenchException, instead of allowing ErrorResponse to generate the random UUID? Would we like this functionality outside of things that extend WorkbenchException?

Reply:
`ErrorResponse` is swagger-generated, so no touchies.

Reply:
Gotcha... and noticed the Who dat?... Geaux Saints!